docker/Dockerfile

```diff
@@ -227,6 +229,10 @@ RUN if [ -z "$PYTORCH_NIGHTLY_URL" ] ; then \
             torchvision==${TORCHVISION_VERSION}+${CUDA_VERSION_TAG} \
             torchtext==${TORCHTEXT_VERSION} ; \
     else \
+        wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.0-1_all.deb && \
+        dpkg -i cuda-keyring_1.0-1_all.deb && \
+        apt-get update && \
+        apt install -y --allow-change-held-packages libnccl2=${NCCL_VERSION}-1+cuda12.0 libnccl-dev=${NCCL_VERSION}-1+cuda12.0 && \
```

Comment:
we will need this for the released version as well. It probably needs to be in a separate block that's something like:

`RUN if [ -n "$NCCL_VERSION" ]`